common/plugins/org.polarsys.capella.common/src/org/polarsys/capella/common/helpers/Triplet.java

```diff
@@ -0,0 +1,89 @@
+package org.polarsys.capella.common.helpers;
```

Comment:
To delete, this class is defined in org.polarsys.capella.common.helpers.cache
Getfirst methods shall be moved from here

Reply:
Done